manager/src/main/java/com/redhat/service/bridge/manager/BridgesServiceImpl.java

```diff
@@ -54,6 +70,22 @@ public Bridge createBridge(String customerId, BridgeRequest bridgeRequest) {
         bridge.setSubmittedAt(ZonedDateTime.now(ZoneOffset.UTC));
         bridge.setCustomerId(customerId);
         bridgeDAO.persist(bridge);
+
+        if (rhoasEnabled) {
```

Comment:
@kostola Under what circumstances would we want to disable the creation of the Topic for an OpenBridge Instance?

Reply:
In my idea, I made it opt-in via a boolean flag property because, since the other tasks regarding interaction with Managed Services are not yet fully implemented (e.g. the deletion of the topic and service account when a bridge is deleted), I assumed that it's better for who handles the deployment to be aware that this (currently incomplete) feature must be explicitly enabled.
Once the whole integration will be completed, we can remove the opt-in parameter.
@robpblake what do you think?

Reply:
@kostola OK, thanks for the explanation. I'm fine with that, but can you please add another JIRA issue to the epic to ensure we remove the feature flag when this work is complete. Worst case is that we leave the property hanging around.

Perhaps also a naming strategy to indicate it's a feature flag e.g `event-bridge.feature-flags.rhoas-enabled=true` and a brief addition to the README so that everyone knows? Might be worth discussing this briefly with the team so that everyone can be aware of this is how we feature flag things on/off.

Reply:
@kostola Feature flagging like this would also be part of the proposal that I've made in our Google chat room.

Reply:
I renamed the property and added a paragraph regarding feature flags to the main `README.md` :+1: